### ISSUE TITLE
if set isInitiallyExpanded: true , then user need to click 2 times to toggle back to shrink mode

### ISSUE THREAD

**Issue by adarshwebcastle** [BinaryBush/toggle_list#1]
### What did you expect to happen?

```
ToggleListItem(
  isInitiallyExpanded: true,
```

First it will expanded by default, and  when user click on item it should shrink/collapse

### What did actually happen?

If I set 
```
ToggleListItem(
  isInitiallyExpanded: true,
```

It's expanding children,  but when user press for the first time nothing happens[not collapsing] ,  but second time only it shrink from expanded mode to collapsed mode. After that it working as expected as usual but only for the first time this issue happens

if I set   isInitiallyExpanded: false it working normally

### Help us recreate the bug.

ToggleListItem(
  isInitiallyExpanded: true,
```

**Comment by sarvarimark** [comment]
Thanks for the bugreport, it's appreciated!
We were able to reproduce the issue based on your description and we found the underlying problem causing the bug. We may need a few days to properly design and implement the solution, but it should be released soon.

**Comment by sarvarimark** [comment]
It turned out to be a bigger issue than expected. To fix it we need to have a deeper inspection of the current architecture and the state related responsibilitites within.
Thank you for your patience in the meantime!

**Comment by adarshwebcastle** [comment]
Thank you for the update. I completely understand the need for a thorough inspection, I'm more than willing to assist in any way I can during this process.  
Your diligence in addressing this matter is much appreciated.

**Comment by sarvarimark** [comment]
The referenced commit, which is released under v0.3.1 should solve the issue.
Thank you for the bug report, for your patience and feel free to reopen if the bug persists.

**Comment by adarshwebcastle** [comment]
Hi, This fix is solved if ToggleListItem has only one item.
If ToggleListItem have more than one item (example 2 items) then I need to click the first item 2 times to shrink it back,
another bug I found that if I try to shrink the second item then the first item also shrink back.


https://github.com/BinaryBush/toggle_list/assets/110030709/a591a6f8-2119-41bf-b181-1217612bbc4b


https://github.com/BinaryBush/toggle_list/assets/110030709/de709556-f48c-46df-b0d1-2893ba86c712
